libs/site-base/src/components/MainPageTransition/styles.module.less

```diff
@@ -0,0 +1,8 @@
+.pageMainInner {
+  min-height: 250px;
```

Comment:
I don't think any of this is necessary, should probably remove it.  It is set by the client apps as well.